.github/workflows/clients.yml

```diff
@@ -0,0 +1,109 @@
+name: Clients
```

Comment:
This is intentionally a single file now, it shouldn't get larger, and there shouldn't be any differences in logic between the clients.